x-pack/plugins/apm/public/plugin.ts

```diff
@@ -44,6 +43,7 @@ import type {
 } from '../../triggers_actions_ui/public';
 import { registerApmAlerts } from './components/alerting/register_apm_alerts';
 import { featureCatalogueEntry } from './featureCatalogueEntry';
+import { createCallApmApi } from './services/rest/createCallApmApi';
```

Comment:
Seems this import is increasing the apm bundle size quite a bit. Any reason it can't still be imported async on line 143?